CMSgov/mdct-mcr
# Auto assign PR creator to PR

## Description
Saves us one step of assigning self - runs a GH action to auto-assign the creator.

### How to test
Open a PR with these changes. Let it assign you. You can see below where it added me (github-actions bot)

### Changed Dependencies
Collect your one extra click per PR

## Code author checklist

## Reviewer checklist (two different people)
